HALO/main.yaml

```diff
@@ -20,6 +20,13 @@ sources:
     driver: yaml_file_cat
     metadata: {}
 
+  SMART:
+    args:
+      path: "{{CATALOG_DIR}}/SMART.yaml"
```

Comment:
below, it seems that you named the file `smart.yaml`, not capitalized. Does this cause the problem? Sorry, I didn't have a closer look so far

Reply:
Good catch! This will certainly be a problem. Macs often don't case about the case of filenames, so it might not be a problem on some computers. Linux generally cares about the case, so it will be a problem on other computers.

Reply:
Ah true okay lets try with both consistent.